BSIS-410 As a user I need to be able to record the venue where I deferred a donor prior to a DIN being issued

timler opened the pull request:
See user story in JIRA: https://jembiprojects.jira.com/browse/BSIS-410

debonair commented:
@timler the build is broken. it seems that its trying to update the venue ids before the column is added do DonorDeferrals

timler commented:
@debonair Sorry I just saw this now. I'm not sure how that's possible because the I made the changesets separate to avoid the problem where I was referencing venue_id column in DonorDeferral before it was created & committed. I have a look now on a clean database.

timler commented:
@debonair I have fixed the issue. It worked fine on a database that already existed, but failed on a new database because the changeset to change location_id to venue_id (in Donor) was done later (in db-after-data.xml). I just moved the changeSets. See 1bc943e
